Controller/NotifyHandler.php

```diff
@@ -182,7 +182,15 @@ private function _onNotificationSuccess($oResponse, $oBackendService)
         }
 
         $oOrder = oxNew(Order::class);
-        if (!$oOrder->loadWithTransactionId($oResponse->getParentTransactionId())) {
+
+        $sTransactionId = $oResponse->getParentTransactionId();
+
+        // Ratepay Invoice and Payolution Invoice do not have a  parent transaction ID set
+        if (is_null($sTransactionId)) {
```

Comment:
Is that the handling of knowing which transactions belong together? If so its much better to send yourself the order id over custom fields or over the url parameters as you will always get them returned in each transaction also post processing transactions.

Reply:
we can do that later. It should not be part of this PR because it'll need a lot of refactoring.

Reply:
will be done in https://jira.parkside.at/browse/WDPD-188